yarn-project/archiver/src/archiver/archiver.ts

```diff
@@ -277,7 +221,7 @@ export class Archiver implements ArchiveSource {
       this.publicClient,
       this.rollupAddress,
       blockUntilSynced,
-      lastL1Blocks.addedBlock + 1n,
+      lastL1Blocks.blocks + 1n,
```

Comment:
the `blocks` here make it sound like a list or so, it reads weirdly with the + 1 on something that looks like a list from the name.